scripts/exrrfs_save_fcst.sh

```diff
@@ -26,13 +18,8 @@ echo "forecast length for this cycle is ${fcst_len_hrs_thiscyc}"
 #
 # decide the location of run and umbrella
 #
-if [[ -z "${ENS_INDEX}" ]]; then
-  ensindexstr=""
-else
-  ensindexstr="/mem${ENS_INDEX}"
-fi
-workdir="${DATAROOT}${ensindexstr}/${RUN}_fcst_${cyc}"
-umbrelladir="${UMBRELLA_DATA}/${jobid}"
+workdir="${DATAROOT}/${RUN}_fcst${MEMID}_${cyc}"
```

Comment:
This was not introduced by this PR but it will be good if we can fix this as early as possible.

We should not define a new variable `workdir`. The NCO implementation standard already has a variable `DATA` which means the same thing as `workdir`.
Also, `DATA` is and should be defined in the J-job:
`export DATA=${DATA:-${DATAROOT}/${jobid}}`
In NCO, it takes the form of `${DATAROOT}/${jobid}`.

Reply:
Can change it to:
workdir="${DATA}"
for now.